### DISCUSSION TITLE
mqtt rpi home assistant

### DISCUSSION THREAD

**Discussion by larschri15** [SBFspot/SBFspot#644] [General]
how can i configure my mqtt broker and set up my config of sbfspot properly so i can read the data? there is no documentation for mqtt on a home assistant instalation the mqtt manual is not usable for home assistant i think (i am still quite new)
I  installed the sbfspot add-on already and everything works there just mqtt doesn't and I installed the mqtt broker add-on

**Comment by jofland** [comment]
You need a mqtt broker like mosquito handling messages between sbfspot and ha.
